### PR TITLE
Added tests for PYTHON-406. NamedTable tests with materialized views.

### PR DESCRIPTION
**Pull request by GregBestland**
This scenario has already be covered traditionally in test_query.py. This tests basically the same functionality using clqengine.

### CODE REVIEW

File: tests/integration/cqlengine/query/test_named.py [hunk 1]
```diff
@@ -259,3 +263,95 @@ def test_get_multipleobjects_exception(self):
...
+        create_mv_alltime = """CREATE MATERIALIZED VIEW {0}.alltimehigh AS
+                        SELECT * FROM {0}.scores
+                        WHERE game IS NOT NULL AND score IS NOT NULL AND user IS NOT NULL AND year IS NOT NULL AND month IS NOT NULL AND day IS NOT NULL
+                        PRIMARY KEY (game, score, user, year, month, day)
+                        WITH CLUSTERING ORDER BY (score DESC)""".format(self.keyspace_name)
+
+        self.session.execute(create_mv_alltime)
+
+        # Populate the base table with data
+        prepared_insert = self.session.prepare("""INSERT INTO {0}.scores (user, game, year, month, day, score) VALUES  (?, ?, ? ,? ,?, ?)""".format(self.keyspace_name))
```
Comment (aholmberg) [inline tests/integration/cqlengine/query/test_named.py]: This block could be tighter and more efficient using the prepared statement and a list of args in  [execute_concurrent_with_args](http://datastax.github.io/python-driver/api/cassandra/concurrent.html#cassandra.concurrent.execute_concurrent_with_args)

File: tests/integration/cqlengine/query/test_named.py [hunk 2]
```diff
@@ -259,3 +263,95 @@ def test_get_multipleobjects_exception(self):
...
+        # Attempt to query the data using Named Table interface
+        # Also test filtering on mv's
+        key_space = NamedKeyspace(self.keyspace_name)
+        table = key_space.table("scores")
+        mv_monthly = key_space.table("monthlyhigh")
+        table_objects = table.objects.all()
+        mv_monthly_objects = mv_monthly.objects.all()
+        mv_all_time = key_space.table("alltimehigh")
+        mv_all_objects = mv_all_time.objects.all()
+        self.assertEqual(len(table_objects), 12)
```
Comment (aholmberg) [inline tests/integration/cqlengine/query/test_named.py]: Possibly base the expected length on the list of input arguments from above.

**Comment by aholmberg** [comment]
Posted some optional take-it-or-leave-it comments.
+1

**Comment by GregBestland** [comment]
Thanks for the feedback, much appreciated. Changes have been made. Merging now.

**Pull request merged**